docker/bragi/Dockerfile

```diff
@@ -39,17 +35,12 @@ RUN --mount=type=cache,target=/home/mimirsbrunn/target \
 # Target image
 # ============
 
-FROM debian:buster-slim
+FROM rust:1.60-alpine3.15
```

Comment:
Would probably be better to use `alpine:3.15` here?

Reply:
why? I prefer to use the image with already rust and cargo rather then install it later

Reply:
That's true for the builder image, but in the target image we don't need Rust installed overall (this is just a compiler, no more).

Reply:
I don't think so if i use an alpine only image I receive this err 

Error loading shared library libgcc_s.so.1: No such file or directory (needed by /usr/bin/bragi)
Error relocating /usr/bin/bragi: _Unwind_Resume: symbol not found